sapbtp-operator-charts/values.yaml

```diff
@@ -93,3 +93,6 @@ externalImages:
       repository: bitnami/kubectl
       sha: ""
       tag: latest
+
+networkPolicy:
```

Comment:
please set a comment with an example for a valid ingress and egress values like [here](https://github.com/SAP/sap-btp-service-operator/blob/main/sapbtp-operator-charts/values.yaml#L55)

Reply:
Done